src/simulators/tensor_network/matrix_product_state.cpp

```diff
@@ -92,6 +92,12 @@ cmatrix_t mul_matrix_by_lambda(const cmatrix_t &mat,
   if (lambda == rvector_t {1.0}) return mat;
   cmatrix_t res_mat(mat);
   uint_t num_rows = mat.GetRows(), num_cols = mat.GetColumns();
+
+  #ifdef _WIN32
+     #pragma omp parallel for
+  #else
+     #pragma omp parallel for collapse(2)
+  #endif
   for(uint_t row = 0; row < num_rows; row++) {
       for(uint_t col = 0; col < num_cols; col++) {
```

Comment:
We need to chage this type too: `uint_t col` => `int_t col`

Reply:
see above.